### ISSUE TITLE
Question in ep. 2 contradicts a previous statement

### ISSUE THREAD

**Issue by ashwinvis** [carpentries-lab/deep-learning-intro#575]
Exercise: The Training Curve: (under [this section](https://carpentries-lab.github.io/deep-learning-intro/2-keras.html#train-model))
> **Do you think the resulting trained network will work well on the test set?**
> Solution: The results of the training give very little information on its performance on a test set. You should be careful not to use it as an indication of a well trained network.

Later on

Exercise: [Confusion Matrix](https://carpentries-lab.github.io/deep-learning-intro/2-keras.html#confusion-matrix)
> **Did you expect this from the training loss you saw?**
> Solution: The training loss was very low, so the low accuracy on the test set may be surprising. But this illustrates very well why a test set is important to give a realistic evaluation when training neural networks (or other machine learning classifiers).

I understand from the solution that you want to emphasize the importance of a test set. However just by reading the question, I was wondering: *Why are we asking this question? Didn't we establish not to connect training loss to the performance of the model *.

We should reformulate the question
